docker/Dockerfile.webapp

```diff
@@ -10,10 +10,10 @@ ENV LANG=C.UTF-8
 # Copy over all Palace build files for this image
 COPY . /ls_build
```

Comment:
Instead of adding docker to the path everywhere, we could change this `COPY` command to be:
`COPY ./docker /lsbuild`

Reply:
i tested. it will not work. it should copy all the files including './docker'.

Reply:
What happens in this case? Why won't it work?

Reply:
the whole project needs to be copied into /lsbuild, not just the docker directory

Reply:
I don't understand why this is the case. We did not copy the whole project into lsbuild before, why is it needed now?

Reply:
ok. i will do double check if i miss anything

Reply:
seems working now.